components/l10n/common/ofac_sanctioned_iso_3166_1_country_code_constants.h

```diff
@@ -23,36 +24,57 @@ inline constexpr auto kOFACSactionedISO31661CountryCodes =
 
         // ISO 3166-1 alpha-2 country codes. See
         // https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2.
+        "AL",  // Albania
+        "BA",  // Bosnia and Herzegovina
         "BY",  // Belarus
         "CU",  // Cuba
         "IR",  // Iran
         "KP",  // North Korea
         "MD",  // Moldova
+        "ME",  // Montenegro
+        "MK",  // North Macedonia
+        "RS",  // Serbia
         "RU",  // Russia
         "SY",  // Syria
         "UA",  // Ukraine (includes Crimea, Donetsk, and Luhansk regions)
+        "VE",  // Venezuela
+        "XK",  // Kosovo
 
         // ISO 3166-1 alpha-3 country codes. See
         // https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3.
+        "ALB",  // Albania
+        "BIH",  // Bosnia and Herzegovina
         "BLR",  // Belarus
         "CUB",  // Cuba
         "IRN",  // Iran
+        "KOR",  // North Korea
         "MDA",  // Moldova
-        "PRK",  // North Korea
+        "MNE",  // Montenegro
+        "MKD",  // North Macedonia
+        "SRB",  // Serbia
         "RUS",  // Russia
         "SYR",  // Syria
         "UKR",  // Ukraine (includes Crimea, Donetsk, and Luhansk regions)
+        "VEN",  // Venezuela
+        "XXK",  // Kosovo
```

Comment:
I can't find `XXK` here: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3.

Reply:
This is preliminary as not defined as of yet. And this is the best match on various sites. Thanks

Reply:
Updated PR with additional Wiki links, thanks